clash-prelude/src/Clash/Class/Counter/Internal.hs

```diff
@@ -0,0 +1,223 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
No type families here either

Reply:
:family:

Reply:
Without it, you can't write `1 <= n` as a class constraint. The other file doesn't need it anymore though!